### PR TITLE
fix(config): Wait 30s for browser activity per Travis.

### PR DESCRIPTION
**Pull request by johnjbarton**
https://docs.travis-ci.com/user/gui-and-headless-browsers/#karma-and-firefox-inactivity-timeouts

### CODE REVIEW

**Comment by johnjbarton** [comment]
Sorry, I only discovered this when trying to further debug the fails.

**Comment by johnjbarton** [comment]
Yes, 10s was the previous default.  30s is too high for local testing, but CI machines are much slower.

File: lib/config.js [hunk 1]
```diff
@@ -343,7 +343,7 @@ class Config {
     }
     this.browserDisconnectTimeout = 2000
     this.browserDisconnectTolerance = 0
-    this.browserNoActivityTimeout = 20000
```
Comment (zzo) [inline lib/config.js]: the previous default was 20s?

Comment (johnjbarton) [reply lib/config.js]: yea, in #3087 two days ago I set it to 20s. Then I saw the travis page recommending 30s. bah.

**Pull request merged**